### PR TITLE
Add: Timeline visualization

### PR DESCRIPTION
**Pull request by androsj**
#### Based on [vis.js](http://visjs.org/docs/timeline/)

Initial implementation that (almost) works!
For some reason it won't draw on initial load; only after toggling filters. I'll keep playing around.

### CODE REVIEW

**Comment by androsj** [comment]
It's working great now! Next, I'll be adding some of its built-in configuration options.

timeline

File: client/app/lib/dynamic-object-access.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import { isNullOrUndefined } from 'util';
+
+// Taken from https://github.com/joshuacc/drabs
+
+/* _obj: the target object.
+   _props: the (nested) target property of _obj. Accepts a string in dot notation or an array of properties.
+   _errorValue: value to return if the property is not found. Defaults to undefined if not provided. */
+export function dynamicGet(_obj, _props, _errorValue) {
+  // Make sure props is defined and not empty
+  if (isNullOrUndefined(_props) || _props.length === 0) {
```
[the remark this answers is not in the document]
Comment (androsj) [reply client/app/lib/dynamic-object-access.js]: Precisely!

[the remark this answers is not in the document]
Comment (kravets-levko) [reply client/app/lib/dynamic-object-access.js]: @arikfr One more reason to replace `underscore` with `lodash` 😃

**Comment by arikfr** [comment]
@androsj considering the significant changes done to visualizations following the React migration, this is no longer viable for merging :-( Thank you for putting in the work and sorry for letting this linger for so long that it became obsolete. :bowing_man:

**Comment by androsj** [comment]
Hey no problem!
There's actually a React component for this, if you are still interested in adding a timeline visualization.

**Comment by arikfr** [comment]
While it might seem otherwise based on the experience here, we're always happy for new visualization types. :grimacing: Will keep this in mind if the need comes up for a timeline.

Thanks!

**Pull request closed without merging**